Source/Fuse.Launcher.InterApp/InterApp/JS.uno

```diff
@@ -13,19 +13,139 @@ namespace Fuse.Reactive.FuseJS
 		You need to add a reference to `"Fuse.Launcher"` in your project file to use this feature.
 
 		## Example
-
+		```javascript
 			var InterApp = require("FuseJS/InterApp");
 
 			InterApp.on("receivedUri", function(uri) {
 				console.log("Launched with URI", uri);
 			});
 
 			InterApp.launchUri("https://fuseopen.com/");
+		```
 
 		In the above example we're using the @EventEmitter `on` method to listen to the `"receivedUri"` event.
 
 		For the [receivedUri](api:fuse/reactive/fusejs/interapp/onreceiveduri_968f99a6.json) event to be triggered, you need register a custom URI scheme for your app, as shown [here](articles:basics/uno-projects#mobile-urischeme).
 
+				
+		## Deep Links - Universal and App Links
+
+		You can receive the `receivedUri` event, mentioned above, for associated web urls that have been tapped on in other apps.
+
+		Apple: [https://developer.apple.com/ios/universal-links](https://developer.apple.com/ios/universal-links)
+
+		Android: [https://developer.android.com/training/app-links](https://developer.android.com/training/app-links)
+
+		## Apple Universal Links
+				
+		1. Add the associated domains in your `.unoproj`
+		2. Add the Apple App Site Association file to your website
+
+		### 1. Add associated domains to your .unoproj
+		```JSON
+		{
+			"iOS": {
+				"SystemCapabilities": {
+					"AssociatedDomains": ["applinks:example.com", "applinks:sub.example.com"]
+				}
+			}
+		}
+		```
+
+		### 2. Add the Apple App Site Association file to your website
+		Apple-app-site-association file format
+		```JSON
+		{
+			"applinks": {
+				"apps": [],
+				"details": [
+					{
+						"appID": "<team identifier>.<bundle identifier>",
+						"paths": [<paths>]
+					}
+				]
+			}
+		}
+		```
+
+		Basic example, this allows all urls of the domain to be validated:
+		```JSON
+		{
+			"applinks": {
+				"apps": [],
+				"details": [
+					{
+						"appID": "1234567890.com.mypackage.myapp",
+						"paths": ["*"]
+					}
+				]
+			}
+		}
+		```
+
+
+		Place this file either in your site’s .`well-known` directory, or directly in its root directory. If you use the `.well-known` directory, the file’s URL should match the following format:
```

Comment:
Do you mean "`.well-known`" instead of ".`well-known`" (place the dot inside the code-segment)?

Reply:
Ha, that's Apple's typo, good spot...

Reply:
Apple never fail to impress. ;)